frontend/src/store/last-will/reducer.ts

```diff
@@ -54,6 +54,18 @@ export const lastWillReducer = (state: LastWill, action: LastWillActions): LastW
 				},
 			}
 		}
+
+		case 'ADD_UNIQUE_PROGRESS_KEY': {
+			if (state.common.progressKeys.find((key) => key === action.payload.progressKey)) return state
```

Comment:
Includes passt denke ich besser: https://www.w3schools.com/jsref/jsref_includes.asp
```suggestion
			if (state.common.progressKeys.includes(action.payload.progressKey)) return state;
```